fixtures/cameo/root-par-6.json

```
@@ -0,0 +1,753 @@
+          "comment": "Magenta"
+        },
+        {
+          "dmxRange": [94, 101],
+          "type": "ColorPreset",
+          "comment": "Pink"
+        },
+        {
+          "dmxRange": [102, 109],
+          "type": "ColorPreset",
+          "comment": "Warm White"
+        },
+        {
+          "dmxRange": [110, 117],
+          "type": "ColorPreset",
+          "comment": "White"
+        },
+        {
+          "dmxRange": [118, 125],
+          "type": "ColorPreset",
+          "comment": "Cold White"
+        },
+        {
+          "dmxRange": [126, 201],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [202, 207],
+          "type": "ColorPreset",
+          "comment": "User Color_1"
+        },
+        {
+          "dmxRange": [208, 213],
+          "type": "ColorPreset",
+          "comment": "User Color_2"
+        },
+        {
+          "dmxRange": [214, 219],
+          "type": "ColorPreset",
+          "comment": "User Color_3"
+        },
+        {
+          "dmxRange": [220, 225],
+          "type": "ColorPreset",
+          "comment": "User Color_4"
+        },
+        {
+          "dmxRange": [226, 255],
+          "type": "ColorPreset",
+          "comment": "UV"
+        }
+      ]
+    },
+    "Amber": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "UV": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "UV"
+      }
+    },
+    "Strobe 2": {
+      "name": "Strobe",
+      "capabilities": [
+        {
+          "dmxRange": [0, 5],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [6, 255],
+          "type": "StrobeSpeed",
+          "speedStart": "1Hz",
+          "speedEnd": "20Hz"
+        }
+      ]
+    },
+    "Dimmer 4": {
+      "name": "Dimmer",
+      "capability": {
+        "type": "Intensity",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Dimmer fein": {
+      "capability": {
+        "type": "Intensity"
+      }
+    }
+  },
+  "modes": [
+    {
+      "name": "3-Channel",
+      "shortName": "3ch",
+      "channels": [
+        "Dimmer",
+        "Color Presets",
+        "DMX Delay"
+      ]
+    },
+    {
+      "name": "5-Channel #1",
+      "shortName": "5ch#1",
+      "channels": [
+        "Dimmer 2",
+        "Strobe Multifuntional",
+        "Color Presets",
+        "Sound Sensitivity",
+        "DMX Delay"
+      ]
+    },
+    {
+      "name": "5-Channel #2 (compatibility for ROOT PAR 4)",
+      "shortName": "5ch#2",
+      "channels": [
+        "Red",
+        "Green",
+        "Blue",
+        "White",
+        "DMX Delay"
+      ]
+    },
+    {
+      "name": "6-Channel",
+      "shortName": "6ch",
+      "channels": [
+        "Dimmer 3",
+        "Strobe",
+        "Chase",
+        "Color Presets 1",
+        "Color Presets 2",
+        "DMX Delay"
```

I hope two Channels with the same functionality works fone with the schema?

```suggestion
        "Dimmer",
        "Strobe",
        "Chase",
        "Color Preset 1",
        "Color Preset 1",
        "DMX Delay"
```

---

See https://github.com/OpenLightingProject/open-fixture-library/pull/1903#discussion_r653402549